ProcessLib/Process.h

```diff
@@ -111,6 +135,19 @@ class Process
 			std::move(config)));
 	}
 
+private:
+	/// Creates mesh subsets, i.e. components, for given mesh.
+	void initializeMeshSubsets()
+	{
+		// Create single component dof in every of the mesh's nodes.
+		_mesh_subset_all_nodes =
+		    new MeshLib::MeshSubset(_mesh, &_mesh.getNodes());
```

Comment:
Agree with transformation. Note taken for later.
Making it unique_ptr requires changes in the dof table...